overlays/libcamera-apps.nix

```diff
@@ -1,28 +1,12 @@
-{ libcamera-apps-src
-, lib
-, stdenv
-, fetchFromGitHub
-, fetchpatch
-, meson
-, pkg-config
-, libjpeg
-, libtiff
-, libpng
-, libcamera
-, libepoxy
-, boost
-, libexif
-, ninja
-}:
-
+{ libcamera-apps-src, lib, pkgs, stdenv }:
```

Comment:
well, those are coming from pkgs, Its mentally easier to wrap your head around the fact that callPackage just provides pkgs rather than it also does a spread, which confuses me, because we're not passing all these extra params.